charts/midaz/templates/ledger/configmap.yaml

```
@@ -33,6 +33,11 @@ data:
   PLUGIN_AUTH_ENABLED: {{ .Values.ledger.configmap.PLUGIN_AUTH_ENABLED | default "false" | quote }}
   PLUGIN_AUTH_HOST: {{ .Values.ledger.configmap.PLUGIN_AUTH_HOST | default "" | quote }}
 
+  # =============================================================================
+  # MULTI-TENANT
+  # =============================================================================
+  MULTI_TENANT_ENABLED: {{ .Values.ledger.configmap.MULTI_TENANT_ENABLED | default "false" | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)